### PR TITLE
SDT-1014 Add html5shiv.js to allow the Account Header to function in IE8

### PR DESCRIPTION
**Pull request by gordonmcmullan**
The header include was missing the code to conditionally add the html5shiv.js script for IE8 and below
(also updated nodemon to avoid the flatmap issue) 
## Before
screenshot from 2019-02-07 14-53-37

## After
screenshot from 2019-02-13 13-03-50
screenshot from 2019-02-13 13-04-37

### CODE REVIEW

File: app/views/layouts/includes/head.html [hunk 1]
```diff
@@ -1,2 +1,5 @@
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
```
[the remark this answers is not in the document]
Comment (rpowis) [reply app/views/layouts/includes/head.html:1]: and even that's a duplicate of the layout _it_ depends on https://github.com/alphagov/govuk-frontend/blob/master/src/template.njk#L14 🤦‍♂️

[the remark this answers is not in the document]
Comment (gordonmcmullan) [reply app/views/layouts/includes/head.html:1]: It's not in the version of govuk-frontend we're using, we could fix it by bumping the version, then we wouldn't need to add this line

[the remark this answers is not in the document]
Comment (gordonmcmullan) [reply app/views/layouts/includes/head.html:1]: And our Account Header layout doesn't depend on _generic.njk

**Comment by gordonmcmullan** [comment]
Will be merged into a larger fix for all dependencies

**Pull request closed without merging**